polyglot_download.py

```diff
@@ -0,0 +1,20 @@
+from polyglot.downloader import downloader
+
+download_dir = '/root/polyglot_data'
+"""
```

Comment:
Put these in the `download.py` script as commented above, I want a single point of entry configured with a single JSON coming from outside.

Also, keep the `embeddings` list as `allowed_embeddings` and you'd do `assert x in allowed_embeddings` to verify the input JSON when downloading.